hidden/Common/Util.swift

```diff
@@ -152,9 +165,10 @@ class Util {
         return names.count >= 2
     }
     
-    static func showPrefWindow() {
+    static func showPrefWindow() -> NSWindow {
         let prefWindow = PreferencesWindowController.shared.window!
         Util.bringToFront(window: prefWindow)
+        return prefWindow
```

Comment:
If this one is Show, it means this function responsibility is show Window.
If you want it to return Window you should named it like getPrefWindow()